searchcore/src/vespa/searchcore/proton/common/replay_feedtoken_state.h

```diff
@@ -0,0 +1,24 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "feedtoken.h"
+#include <vespa/vespalib/util/shared_operation_throttler.h>
+
+namespace proton::feedtoken {
+
+/*
+ * Feed token state used during replay.
```

Comment:
Consider adding a comment regarding the wrapping of a throttler token, and what this implies.

Reply:
Done